GETTING_STARTED.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started
+
+## Installing Requirements
+1. You can clone the Github project by typing in ````git clone git@github.com:lhain08/FitbitDashboard.git```` in the command line in order to obtain a local copy of the project 
+2. This project contains the [python-fitbit](https://github.com/orcasgit/python-fitbit/tree/6a0a7cba26c26e6c8096bf51d4cf7f19e113ed96) Github project as a submodule. However, after cloning this repository, you will see that in ````src/api/python_fitbit```` is empty in your local repo. To obtain the files from the project, run this in your command line: 
+
+   ````git submodule update --init --recursive````
+
+3. Navigate to ````dev.txt```` via ````src/api/python_fitbit/requirements````. Open the text file, and you will see 1 line that reads: ````cherrypy>=3.7,<3.9````. Delete this line. 
```

Comment:
Before we close the issue this is addressing, lets try and phase this step out. We should check the licensing on python-fitbit and see if we can just copy what we need from that repo into our project, make the fix we need to the requirements, and cut out anything we don't require.